layouts/partials/head.html

```diff
@@ -232,7 +232,7 @@
     </script>
 
     <script async defer src="https://buttons.github.io/buttons.js"></script>
-    {{- if eq hugo.Environment "production" }}
+    {{- if ne hugo.Environment "production" }}
```

Comment:
This is just so we can validate this change in the PR preview. We'll switch it back before merging.